src/index.js

```diff
@@ -1,3 +1,137 @@
+// 전역 변수
+const todoItem = `
+<div class="view">
+  <input class="toggle" type="checkbox"/>
+  <label class="label">새로운 타이틀</label>
+  <button class="destroy"></button>
+</div>
+<input class="edit" value="새로운 타이틀" />
+`;
+let state = {
+    condition: "all",
+    items: [],
+};
+const todoList = document.getElementById("todo-list");
+
+// state객체의 상태에 따라 새로고침 해주는 메소드
+// state.condition 에 따라 각각 다른 리스트를 보여준다.
+// 함수의 마지막에 localStorage에 저장하는 코드가 있음
+function rePaint(state) {
+    todoList.innerHTML = '';
+    let rePaintList = [];
+    if (state.condition === "all") rePaintList = state.items; 
+    if (state.condition === "active") rePaintList = state.items.filter(item=> item.completed === false);
+    if (state.condition === "completed") rePaintList = state.items.filter(item=> item.completed === true);
```

Comment:
```suggestion
    const { condition, items } = state;
    rePaintList = items.filter(item => (condition === 'all') ||
                                       (condition === 'completed' && item.completed) ||
                                       (condition === 'active' && !item.completed))
```
이렇게 표현할 수 있습니다.
condition의 상태는 constant로 관리해보는건 어떨까요?
```js
const filerTypes = { ALL: 'all', COMPLETED: 'completed', ACTIVE: 'active' }
```
이렇게 변수를 만들어서 관리할 경우 코드 추적하기도 쉽고, 변경사항에 대응하기도 쉽습니다!